### PR TITLE
Enable RBAC for multi DiscoveryConfigs

### PR DESCRIPTION
**Pull request by zkayyali812**
https://github.com/open-cluster-management/backlog/issues/11500

### CODE REVIEW

File: frontend/public/locales/en/discovery.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
     "discovery.edit": "Edit cluster discovery",
     "discovery.import": "Import cluster",
 
-    "disable.title": "Disable cluster discovery",
-    "disable.button": "Disable",
-    "disable.message": "You are disabling all cluster discovery. All connections will be removed from the filter, and the filter and current discovered clusters will be deleted.",
+    "disable.title": "Delete discovery config",
```
Comment (swopebe) [inline frontend/public/locales/en/discovery.json:6]: In these, should "discovery config" be "discoveryConfig" and if not, consider a full phrase: "discovery configuration" please?

File: frontend/public/locales/en/discovery.json [hunk 2]
```diff
@@ -3,9 +3,9 @@
...
+    "discoveryConfig.delete.btn": "Delete",
+    "discoveryConfig.delete.message": "You are deleting discovery config <bold>{{discoveryConfigName}}</bold>. All related discovered clusters will be deleted.",
```
Comment (swopebe) [inline frontend/public/locales/en/discovery.json:9]: same here

File: frontend/public/locales/en/discovery.json [hunk 3]
```diff
@@ -3,9 +3,9 @@
...
```
Comment (swopebe) [inline frontend/public/locales/en/discovery.json:8]: Recommend: You are deleting the <bold>{{discoveryConfigName}}</bold> discovery configuration.

**Review by swopebe — CHANGES_REQUESTED** [review]
See comments

File: frontend/public/locales/en/discovery.json (hunk 3 above)
Comment (zkayyali812) [reply frontend/public/locales/en/discovery.json]: updated

**Review by jamestalton — APPROVED** [review]
lgtm

**Review by cameronmwall — APPROVED** [review]

**Review by nathanweatherly — APPROVED** [review]

**Pull request merged**